pkg/util/globalerror/errors.go

```diff
@@ -61,16 +61,16 @@ const (
 
 	StoreConsistencyCheckFailed ID = "store-consistency-check-failed"
 	BucketIndexTooOld           ID = "bucket-index-too-old"
+
+	MsgSizeTooLarge ID = "msg-size-too-large"
```

Comment:
Make it specific for the distributor, since the runbook is specific.

```suggestion
	DistributorMaxWriteMessageSize ID = "distributor-max-write-message-size"
```